engine/test-src/org/pentaho/di/repository/bad-repositories.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Should we move the XML files to test-res/ ? Or is it called testfiles/ in kettle?